CHANGELOG.md

```diff
@@ -10,7 +10,7 @@ This project adheres to the Node [default version scheme](https://docs.npmjs.com
 
 ### Added
 
-- UI: Add country selection step after document selection
+- UI: Add country selection screen after document selection. This screen is skipped by default for a preselected document but can still be displayed by enabling the `skipCountrySelection` option for the `document` step.
```

Comment:
It's a bit confusing here: should it be `showCountrySelection`?

Reply:
Yes, you're right. Good spot!